### PR TITLE
feat(webapp): display current distribution rate

### LINKED ISSUE #369: Display current distribution Rate
Display the current distribution rate and expected disbursements for the next month would be great info to have on the site

Image

### PR DESCRIPTION
**Pull request by warnerHurtado**
### What does this PR do?

- Resolve #369 

### Steps to test

1. run the project and  open it in localhost:3000
2. Go to income and see the next distribution amount 

image

### CODE REVIEW

File: webapp/src/language/en.json [hunk 1]
```diff
@@ -83,6 +83,14 @@
...
     "claimedCategory": "Claimed",
     "unclaimedCategory": "Unclaimed",
+    "titleDisbursement": "Monthly disbursements",
+    "paragraph1Disbursement": "Eden delegate disbursements occur monthly and are claimed by the delegate from the contract to their personal EOS accounts.",
+    "paragraph2Disbursement": "The overall disbursement is equal to 11% of the Eden treasury at at the time of disbursement. The amount is then divided equally among the representative levels. At each level, the amount is further divided equally among that level's representatives.",
+    "paragraph3Disbursement": "Next disbursement date",
+    "paragraph4Disbursement": "Projected next disbursement amount:",
+    "perLevel": "Per Level 1 Delegate",
+    "perChief": "Per Chief Delegate",
+    "trasury": "Eden Treasury Balance"
```
Comment (xavier506) [inline webapp/src/language/en.json:93]: Projected Treasury Balance

**Comment by xavier506** [comment]
There is only Level 1 Delegates and Chief Delegates on Eden right now @warnerHurtado

File: webapp/src/language/en.json [hunk 2]
```diff
@@ -83,6 +83,14 @@
     "tableHeader8": "Unclaimed %",
     "claimedCategory": "Claimed",
     "unclaimedCategory": "Unclaimed",
+    "titleDisbursement": "Monthly disbursements",
+    "paragraph1Disbursement": "Eden delegate disbursements occur monthly and are claimed by the delegate from the contract to their personal EOS accounts.",
+    "paragraph2Disbursement": "The overall disbursement is equal to 11% of the Eden treasury at at the time of disbursement. The amount is then divided equally among the representative levels. At each level, the amount is further divided equally among that level's representatives.",
+    "paragraph3Disbursement": "Next disbursement date",
+    "paragraph4Disbursement": "Projected next disbursement amount:",
+    "perLevel": "Per Level 1 Delegate",
+    "perChief": "Per Chief Delegate",
```
Comment (xavier506) [inline webapp/src/language/en.json:92]: Per Chief Delegate (L1 + L2)

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Pull request merged**